integration/windows_hostprocess_test.go

```diff
@@ -124,3 +124,35 @@ func runHostProcess(t *testing.T, expectErr bool, image string, action hpcAction
 
 	action(t, cn, containerConfig)
 }
+
+func TestArgsEscapedImagesOnWindows(t *testing.T) {
+	containerName := "test-container"
+	testImage := images.Get(images.ArgsEscapedTestImage)
+	sb, sbConfig := PodSandboxConfigWithCleanup(t, "sandbox", testImage, WithWindowsHostProcessPod)
+	EnsureImageExists(t, testImage)
+
+	cnConfig := ContainerConfig(
+		containerName,
+		testImage,
+		WithCommand("ping", "-t", "127.0.0.1"),
```

Comment:
Should we have a default test case with no cmd merge?

Reply:
Done